### PR TITLE
feat: adding source

### PR DESCRIPTION
**Pull request by richweclg**
@loahou04 @TanujaG @marcdonovan

### CODE REVIEW

File: src/js/NotificationNode.js [hunk 1]
```diff
@@ -24,11 +24,11 @@ export default class NotificationNode extends React.Component {
 						<div className="notification-node--summary">
 							<h1> {this.props.title} </h1>
 							<div className="notification-node--summary-description">{this.props.summary}</div>
-							
+
 						</div>
 						<div className="notification-node--meta">
 							<div className="notification-node--meta-course">
-								{this.props.time} &#183; US History
+								{this.props.time}{this.props.source ? ' · ' : ''}{this.props.source}
```
Comment (loahou04) [inline src/js/NotificationNode.js]: o you found the actual character on the keyboard for it?  i used the html encoded character instead.  Where is this located on the keyboard?

**Comment by loahou04** [comment]
approved

File: src/js/NotificationNode.js (hunk 1 above)
Comment (richweclg) [inline src/js/NotificationNode.js]: I did a google search for the html encoded character and copied it from one of the results :-)

Comment (loahou04) [inline src/js/NotificationNode.js]: ok well maybe we should leave it as the html encoded character then?

**Comment by TanujaG** [comment]
approved

**Pull request merged**